listing.php

```diff
@@ -191,22 +191,166 @@ function showDirFiles($svnrep, $subs, $level, $limit, $rev, $peg, $listing, $ind
 	return $listing;
 }
 
-function showTreeDir($svnrep, $path, $rev, $peg, $listing) {
-	global $vars, $config;
+function showAllDirFiles($svnrep, $path, $rev, $peg, $listing, $index, $treeview = true) {
```

Comment:
Changed the treeview to treeView. Yeah i will keep this in mind as well. As i took this piece of code from existing code. But you are right need to improve when writing something new otherwise code style will never improve.